activities/Stopwatch.activity/js/activity.js

```diff
@@ -263,4 +263,4 @@ define(["sugar-web/activity/activity","mustache", "sugar-web/env"], function (ac
 
     });
 
-});
```

Comment:
No need to make a change where not required.

Reply:
There was only a removal of new line character. Fixed it.